Tests/AK/TestOptionParser.cpp

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+
+#include <AK/Array.h>
+#include <AK/OptionParser.h>
+#include <AK/String.h>
+#include <AK/Vector.h>
+
+TEST_CASE(string_option)
+{
+    AK::ByteString short_options = "";
```

Comment:
Please don't qualify entities in the `AK` ns unless a conflict exists.
Also, ByteString's default state is the empty string, so:
```suggestion
    ByteString short_options;
```